script/i18n.coffee

```diff
@@ -15,10 +15,13 @@ module.exports =
 
     locale = 'en' if (!locale? or (!module.exports.strings and !module.exports.translations[locale]))
     string = if (!module.exports.strings) then module.exports.translations[locale][stringName] else module.exports.strings[stringName]
+    
+    clonedVars = _.clone(vars) or {};
```

Comment:
hm what do you mean? we should I do it then?

I thought If I not cloned it, then we could also get cross-request of different languages